packages/sui-domain/src/AnemicModel.js

```diff
@@ -1,3 +1,28 @@
+/**
+ * Recursively calls toJSON on each member
+ * @param {Object} obj
+ */
+const toJSON = obj => {
```

Comment:
The naming of toJSON could lead us to missunderstand what are we calling. We could think that we are calling `obj.toJSON()` instead that call a local toJSON function.

Reply:
I'm assuming here that the obj was created brand new because it's the same scope (which is the case).
But I agree I could be more functional. Will change it. Thanks.